manager-parent/manager-v2/src/main/java/com/redhat/service/smartevents/manager/v2/services/BridgeServiceImpl.java

```diff
@@ -232,53 +238,62 @@ public Bridge updateBridgeStatus(ResourceStatusDTO statusDTO) {
             return bridge;
         }
         Operation operation = bridge.getOperation();
+        if (Objects.isNull(operation)) {
```

Comment:
@manstis According to the DB schema, this can never happen as both columns of the embedded `Operation` object have a `not null` clause on them.

Reply:
Removed check.